04_environment/index.tex

```diff
@@ -1 +1,400 @@
-\chapter{Game Environment}
\ No newline at end of file
+\chapter{Environment}
+\section{Foraging}
+\subsection{Foraging Background}
+
+Foraging is a base requirement that is needed as it is a method of obtaining more resources from the starting amount.Foraging is based on the concept of inputting resources to get a return.\\
+
+\subsubsection{Underlying concept}
+The goal of foraging in this “game” is to introduce a certain dilemma to the agents to further understand the behaviours of multiple (6) intelligent agents working together to reach a goal with different personal goals. By having 2 methods of foraging it introduces the concept of decision making between the islands allowing for strategic interactions to maximise personal benefits whilst still reaching the overarching goal of survival.\\  
+
+The inspiration for our concept of foraging comes from the stag hunt game furthermore it introduces elements from it. Specifically it brings in the concept of \textbf{no dominant strategy} and how \textbf{ one agents choice can impact other agents.}\\
+
+The concept of stag hunt is that each player selects a method of hunting and if they differ then the social welfare (total payoff) is significantly less than if they both go stag hunting. The selection of stag means the agent is going for higher \textbf{payoff}whilst the selection of hare indicates that the agent is going for \textbf{risk aversion} as both agents are required to stag hunt to get any personal welfare. \\
+
+\subsubsection{Modifications to the original concepts} 
+A few changes have to be made to make this dilemma viable for our multi-agent system, in addition to adding realism. 
+
+\begin{enumerate}
+    \item Changed the dependency to be based on the input resources rather than the number of islands participating. This added a more realistic element to the dilemma as in reality a hunt’s return would be based on the amount of resources (people, food, water and materials) entered rather than the number of islands participating.
+    \item Instead of a specified return we have a probabilistic return which adds a randomness element to the game to prevent pattern recognition. These probabilities can be tied to the population, disasters, forecasting and number of animals hunted.  
+\end{enumerate}
+
+\newpage
+\subsection{Functions within Foraging}
```

Comment:
can you remove this subsection from this PR? :)

Reply:
Hi, This section is a part of the thought process of how a key element of the game is run. 
Why do you want us to remove this section @ezgimez?

Reply:
I think we resolved this conflict :) in our talk Catherine